Project.toml

```diff
@@ -11,6 +11,7 @@ Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 
 [compat]
+Distributions = "0.22.3, 0.23, =0.24.0, =0.24.1, =0.24.2, =0.24.3, =0.24.4, =0.24.5, =0.24.6, =0.24.7, =0.24.8, =0.24.9, =0.24.10, =0.24.11"
```

Comment:
Can we not do this?

```suggestion
Distributions = "0.22.3, 0.23, <0.24.12"
```

Reply:
That's equivalent to just writing: `"<0.24.12"`. Definitely is a viable alternative but doesn't let us set a minimum

Reply:
Ah... and hyphen separators are only Julia 1.4+.

LGTM in this case! When new LTS comes in we can clean this up I suppose.